openthread/cmake/extensions.cmake

```diff
@@ -26,7 +50,61 @@ endif()
   endforeach()
 
   list(SORT OPENTHREAD_SETTINGS)
-  list(INSERT OPENTHREAD_SETTINGS 0 "OpenThread_commit=${openthread_git_hash}\n")
+
+  find_package(Git QUIET)
+  if(GIT_FOUND)
+    get_mbedtls_dir(ARM_MBEDTLS_PATH)
+    get_git_hash(${ARM_MBEDTLS_PATH})
+    list(INSERT OPENTHREAD_SETTINGS 0 "MBEDTLS_commit=${git_hash}\n")
```

Comment:
I know a SHA is unique, but in `west.yml` we prefer to point to a tag.
The same when doing an NCS release, where all Nordic repos are tagged.

So in case the SHA is also a tag, I think it would be nice to include the tag as well.
Makes it easier to find exact version used (and the fact it was a release).
Keep the SHA also.